test/EFCore.PG.FunctionalTests/Query/ArrayListQueryTest.cs

```diff
@@ -843,6 +843,18 @@ WHERE array_cat(s."IntList", ARRAY[5,6]::integer[]) = ARRAY[3,4,5,6]::integer[]
 """);
     }
 
+    public override async Task Remove(bool async)
+    {
+        await base.Concat(async);
```

Comment:
I'm not quite sure what you're trying to do here (i.e. why is test Remove calling base.Concat). You need to actually add a Remove test in the base class (ArrayQueryTest), as well as in ArrayArrayQueryTest.